code/modules/jobs/job_types/assistant/gimmick_assistants.dm

```diff
@@ -0,0 +1,211 @@
+// This file contains the louadouts for the assistant gimmicks station trait! When active, gives assistants random stupid gear
+
+/// Parent type of gimmick loadouts for assistants for the functional assistant station traits
+/datum/outfit/job/assistant/gimmick
+	name = "Gimmick Assistant"
+	/// The weight of the outfit to be picked
+	var/outfit_weight = 0
+
+
+/datum/outfit/job/assistant/gimmick/give_jumpsuit(mob/living/carbon/human/target)
+	return //dont do colorized and stuff, it messes with our uniforms
+
+/datum/outfit/job/assistant/gimmick/bee
+	name = "Gimmick Assistant - Bee"
+	suit = /obj/item/clothing/suit/hooded/bee_costume
+	uniform = /obj/item/clothing/under/color/yellow
+
+	l_pocket = /obj/item/coupon/bee
+
+	outfit_weight = 2
+
+/obj/item/coupon/bee
+	name = "coupon - 70% off Beekeeping Starter Crate"
```

Comment:
This seems like something that should be done automatically by ``update_name``

Reply:
Uh sure, I’ll shake some code around